apollo-router/src/services/layers/persisted_queries/mod.rs

```diff
@@ -110,9 +111,10 @@ impl PersistedQueryLayer {
         } else {
             // if there is no query, look up the persisted query in the manifest
             // and put the body on the `supergraph_request`
-            if let Some(persisted_query_body) =
-                manifest_poller.get_operation_body(persisted_query_id)
-            {
+            if let Some(persisted_query_body) = manifest_poller.get_operation_body(
+                persisted_query_id,
+                request.context.get(CLIENT_NAME).unwrap_or_default(),
```

Comment:
This probably implies that the new feature only works if Apollo telemetry is enabled. This is probably a reasonable constraint for an initial release, but we should at least document it.